Extension/src/LanguageServer/client.ts

```diff
@@ -3515,18 +3514,10 @@ export class DefaultClient implements Client {
                         // Destination file is empty.
                         // The edit positions for #include header file and definition or declaration are the same.
                         selectionPositionAdjustment = (lastEdit.newText.match(/\n/g) || []).length;
```

Comment:
The line `if (lastEdit.range.isEqual(range)) {` can be combined with the previous line into one conditional since the else for `if (lastEdit.range.isEqual(range))` is removed.